1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive = everyone.slice(0.5); // complete this statement
+let lastFive = .slice(-5); // complete this statement
```

Comment:
I'm afraid that this one won't work, needs a comma instead of a dot and the senond needs you to put the name of the array in front of it